config/locales/en.yml

```diff
@@ -43,6 +43,7 @@ en:
   new: New
   save: Save
   close: Close
+  select_team: Please select a team or use the search
```

Comment:
Thinking about it...maybe something less imperative?
I like: _Looking for a password?_ (similar to before, actually)
Of course, if you change it, don't forget to change it for all languages, duh.